### PR TITLE
zigbee: Fix setting intervals between image query

### PR DESCRIPTION
**Pull request by tomchy**
Fix the attribute name - it is setting interval between server queries instead of image queries and add the correct one.

Signed-off-by: Tomasz Chyrowicz <tomasz.chyrowicz@nordicsemi.no>

### CODE REVIEW

**Review by robertstypa — APPROVED** [review]

File: doc/nrf/libraries/zigbee/zigbee_fota.rst [hunk 1]
```diff
@@ -34,8 +34,9 @@ The following entities participate in the Zigbee OTA Upgrade process:
 OTA upgrade process
 *******************
 
+OTA upgrade Client queries for OTA Upgrade servers with intrvals defined by :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` until at least one server is found.
```
Comment (FilipZajdel) [inline doc/nrf/libraries/zigbee/zigbee_fota.rst:37]: intrvals -> intervals

**Review by FilipZajdel — APPROVED** [review]
LGTM. Just one minor typo to correct.

**Review by sebastiandraus — APPROVED** [review]

File: doc/nrf/libraries/zigbee/zigbee_fota.rst (hunk 1 above)
Comment (tomchy) [reply doc/nrf/libraries/zigbee/zigbee_fota.rst]: Fixed

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -769,7 +769,8 @@ Libraries for Zigbee
 
     * Added:
 
-      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig option to configure interval between queries for the available Zigbee FOTA images.
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option to configure interval between queries for the Zigbee FOTA server.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:772]:
```suggestion
      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option to configure the interval between queries for the Zigbee FOTA server.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -769,7 +769,8 @@ Libraries for Zigbee
...
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option to configure interval between queries for the available Zigbee FOTA images.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:773]:
```suggestion
      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option to configure the interval between queries for the available Zigbee FOTA images.
```

File: doc/nrf/libraries/zigbee/zigbee_fota.rst [hunk 4]
```diff
@@ -34,8 +34,9 @@ The following entities participate in the Zigbee OTA Upgrade process:
 OTA upgrade process
 *******************
 
+OTA upgrade Client queries for OTA Upgrade servers with intervals defined by :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` until at least one server is found.
```
Comment (greg-fer) [inline doc/nrf/libraries/zigbee/zigbee_fota.rst:37]:
```suggestion
The OTA Upgrade Client queries for OTA Upgrade Servers with the intervals defined by the :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option until at least one server is found.
```

File: doc/nrf/libraries/zigbee/zigbee_fota.rst [hunk 5]
```diff
@@ -34,8 +34,9 @@ The following entities participate in the Zigbee OTA Upgrade process:
...
+Once found, it will start to query the server for images. The interval between queries for the available Zigbee FOTA images is defined by :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN`.
```
Comment (greg-fer) [inline doc/nrf/libraries/zigbee/zigbee_fota.rst:38]:
```suggestion
Once found, the client starts to query the server for images. 
The interval between queries for the available Zigbee FOTA images is defined by the :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/zigbee/zigbee_fota.rst (hunk 5 above)
Comment (tomchy) [reply doc/nrf/libraries/zigbee/zigbee_fota.rst]: Fixed.

File: doc/nrf/libraries/zigbee/zigbee_fota.rst (hunk 4 above)
Comment (tomchy) [reply doc/nrf/libraries/zigbee/zigbee_fota.rst]: Fixed.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 3 above)
Comment (tomchy) [reply doc/nrf/releases/release-notes-changelog.rst]: Fixed.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (tomchy) [reply doc/nrf/releases/release-notes-changelog.rst]: Fixed.

**Review by greg-fer — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**